FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.HealthCheck/HealthCheckBackgroundService.cs

```diff
@@ -45,42 +49,42 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
         {
             while (!cancellationToken.IsCancellationRequested)
             {
+                // Delay interval time.
+                var delayTask = Task.Delay(_checkIntervalInSeconds, cancellationToken);
                 try
                 {
+                    _diagnosticLogger.LogInformation("Starting to perform health checks");
                     _logger.LogInformation("Starting to perform health checks");
 
                     // Perform health check.
                     var healthStatus = await _healthCheckEngine.CheckHealthAsync(cancellationToken);
 
+                    _diagnosticLogger.LogInformation($"Finished health checks: ${string.Join(',', healthStatus.HealthCheckResults.Select(x => x.Name))}.");
```

Comment:
Will we have a lot of healthcheck logs there?

Reply:
Health check result will not too complicated here.